pkg/ebpf/c/tracee.bpf.c

```diff
@@ -5294,17 +5339,30 @@ int BPF_KPROBE(trace_security_bpf)
     return events_perf_submit(&data, SECURITY_BPF, 0);
 }
 
-SEC("kprobe/arm_kprobe")
-int BPF_KPROBE(trace_arm_kprobe)
+SEC("kprobe/enable_kprobe")
+TRACE_ENT_FUNC(enable_kprobe, KPROBE_ATTACH);
+
+SEC("kretprobe/enable_kprobe")
```

Comment:
@roikol could you give me a +1 here ? I believe it works like the previous way, but now for arm64 as well. Thanks.